extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php

```diff
@@ -386,11 +404,14 @@ public function Phalanx() {
 
 		wfRunHooks( 'CheckContent', array( $text, &$blockedKeyword ) );
 
-		$this->msgHeader = '';
-		$this->msgBody = '';
 		if ( !empty( $blockedKeyword ) ) {
-			$this->msgHeader = wfMsg('cnw-badword-header');
-			$this->msgBody = wfMsg('cnw-badword-msg', $blockedKeyword);
+			$this->info( __METHOD__ . ": keyword blocked by Phalanx '" . $text . "'': " . $blockedKeyword );
```

Comment:
you would still need to concatenate `": keyword blocked by Phalanx '" . $text` here. I think it would be more readable if we could replace concatenation with `implode`. Mixing these two introduces more complexity IMHO